TypeScript Error: AuthInfo type does not exist

**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
I'm using `pusher-js/with-encryption` in a TypeScript project. When I attempt to compile, I get the following error:
```shell
node_modules/pusher-js/with-encryption/index.d.ts:2:3 - error TS2305: Module '"../types/src/core/auth/options"' has no exported member 'AuthInfo'.

2   AuthInfo,
    ~~~~~~~~
```

It looks like the `AuthInfo` type definition was removed in v7.0.0 (see https://github.com/pusher/pusher-js/commit/96aab17f993be499ce1b49ef0649e4f749ac9e81#diff-dbc910045a791a925c9af7c217c3abae2cff6d78d9df1f443188b7103b3cc2b7L10), but it's still being referenced.

**Proposed solution:** Removing the export of `AuthInfo` from https://github.com/pusher/pusher-js/blob/9daa2ee8afcd1ad70e31288652fd24e2ec49c241/with-encryption/index.d.ts#L2 and https://github.com/pusher/pusher-js/blob/8b2c1ba76323abee0ce9f9e75f52767013c27744/react-native/index.d.ts#L2 should resolve the issue.

--- a/react-native/index.d.ts
+++ b/react-native/index.d.ts
@@ -1,16 +1,15 @@
 export {
-  AuthInfo,
   Authorizer,
   AuthOptions,
   AuthorizerGenerator,
   AuthorizerCallback,
 } from '../types/src/core/auth/options';
-export { Options } from '../types/src/core/options'
+export { Options } from '../types/src/core/options';
 
-export {default as Channel} from '../types/src/core/channels/channel';
-export {default as PresenceChannel} from '../types/src/core/channels/presence_channel';
-export {default as Members} from '../types/src/core/channels/members';
-export {default as  Runtime}  from '../types/src/runtimes/interface'
-export {default as ConnectionManager}  from '../types/src/core/connection/connection_manager'
+export { default as Channel } from '../types/src/core/channels/channel';
+export { default as PresenceChannel } from '../types/src/core/channels/presence_channel';
+export { default as Members } from '../types/src/core/channels/members';
+export { default as Runtime } from '../types/src/runtimes/interface';
+export { default as ConnectionManager } from '../types/src/core/connection/connection_manager';
 
-export {default} from '../types/src/core/pusher'
+export { default } from '../types/src/core/pusher';

--- a/with-encryption/index.d.ts
+++ b/with-encryption/index.d.ts
@@ -1,5 +1,4 @@
 export {
-  AuthInfo,
   Authorizer,
   AuthOptions,
   AuthorizerGenerator,

